src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1495,12 +1496,12 @@ def process_image_create_namespace(cmd, namespace):
             compute_client = _compute_client_factory(cmd.cli_ctx, subscription_id=res['subscription'])
             vm_info = compute_client.virtual_machines.get(res['resource_group'], res['name'])
             source_from_vm = True
-    except CloudError:
+    except ResourceNotFoundError:
```

Comment:
cloudError is more than ResourceNotFound 404. what if other http status code?

Reply:
In this method, only ResourceNotFoundError can be ignored. Other errors should be thrown.